app/backend/src/tests/test_interceptors.py

```diff
@@ -57,6 +58,21 @@ def interceptor_dummy_api(
             server.stop(None).wait()
 
 
+def _get_count_from_histogram():
+    metrics = servicer_duration_histogram.collect()
+    servicer_histogram = next(m for m in metrics if m.name == "servicer_duration")
+    return next(s for s in servicer_histogram.samples if s.name == "servicer_duration_count")
```

Comment:
Are these `next()`s the most obvious way of doing this? Are you basically just doing `[comprehension][0]` here?

Reply:
Yeah, I'm not in love with they python syntax here (I am used to being able to do` listOfThings.first( lambda )` ). Happy to change it to `[comprehension][0]`.